ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -382,7 +383,7 @@ def ndmg_dwi_worker(
             np.eye(4),
         )
         streamlines = trct.run()
-        streamlines = Streamlines([sl for sl in streamlines if len(sl) > 40])
+        streamlines = Streamlines([sl for sl in streamlines if len(sl) > 60])
```

Comment:
guessing this is just for fiddling purposes?

Reply:
60 mm minimum streamline length filter is magic.